backend/app/views/admins/sessions/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="bg-white p-6 rounded-2xl shadow-lg min-w-[600px]">
+  <h2 class="text-green-dark font-bold text-lg">Welcome to HeCO Invest backoffice.</h2>
+
+  <p class="my-6 text-gray-600">Please enter your details below.</p>
+
+  <%= simple_form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+    <% flash.each do |type, message| %>
```

Comment:
maybe special partial for flash messages?

Reply:
Does not need it at this moment, maybe when adding forgot password stuff. This is the quirk of devise that is using flash messages, and not kind of form objects with errors collection for error messages.

Reply:
So it is very specific only to this place.